aws/terminator.yml

```diff
@@ -52,34 +52,34 @@
       pip:
         requirements: "{{ playbook_dir }}/requirements.txt"
         virtualenv: "{{ packaging_dir }}/terminator-requirements/python"
-        virtualenv_python: python3.7
+        virtualenv_python: python3.9
```

Comment:
we should rather define this in a dedicated to simplify code updates, I have done it in another PR

Reply:
Oh sorry for the confusion, this PR is not ready for review. It's a series of commits that I want to submit in independent PR.